tests/crc_test.c

```diff
@@ -73,8 +73,9 @@ static int s_test_known_crc_32(
     uint32_t result = func(input, (int)length, 0);
     ASSERT_HEX_EQUALS(expected_crc, result, "%s(%s)", func_name, data_name);
 
+    uint32_t result_swapped = aws_swap_bytes_if_needed_32(result);
```

Comment:
trivial: it's not super obvious which way "swapped" is
```suggestion
    uint32_t result_little_endian = aws_swap_bytes_if_needed_32(result);
```
or
```suggestion
    uint32_t result_le = aws_swap_bytes_if_needed_32(result);
```